src/main/java/frc/robot/swerve/SwerveConfig.java

```diff
@@ -58,48 +49,32 @@ public class SwerveConfig {
     public static final int BR_TURN_CANID = 20;
     public static final boolean BR_REVERSE = false;
 
-    // helper function
-
-
-
-
-
-
-    ///////////////////////////////////////////// UNDERCONSTRUCTION
-
-
-
-
-    //The createDriveMotor method 
-
-    public static void createDriveMotor(int canld, boolean reverseMotor){
-
-        TalonFX DriveMotor = new TalonFX(canld);
-        {
-          DriveMotor.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
-          DriveMotor.configSoftLimitDisableNeutralOnLOS(true, canld);
-          DriveMotor.setInverted(InvertType.None);
-          DriveMotor.setNeutralMode(NeutralMode.Brake);
-          DriveMotor.configClosedloopRamp(0.3);
-          DriveMotor.config_kP(0, 0.1);
-          DriveMotor.config_kI(0, 0.001);
-          DriveMotor.config_kD(0, 5.0);
-          DriveMotor.config_kF(0, 1023.0 / 20660.0);
-          DriveMotor.config_IntegralZone(0, 300.0);
-          DriveMotor.configMaxIntegralAccumulator(0, 1.0);
+    /**
+     * Creates a drive motor for the swerve drive. Drive motors are used with
+     * velocity-based PID control.
+     */
+    public static TalonFX createDriveMotor(int canId, boolean reverseMotor){
+        TalonFX driveMotor = new TalonFX(canId);
+        driveMotor.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
+        driveMotor.configSoftLimitDisableNeutralOnLOS(true, canId);
+        if (reverseMotor) {
+            driveMotor.setInverted(InvertType.None);
+        } else {
+            driveMotor.setInverted(InvertType.InvertMotorOutput);
         }
-  
+        driveMotor.setNeutralMode(NeutralMode.Brake);
+        driveMotor.configClosedloopRamp(0.3);
+        driveMotor.config_kP(0, 0.1);
+        driveMotor.config_kI(0, 0.001);
+        driveMotor.config_kD(0, 5.0);
+        driveMotor.config_kF(0, 1023.0 / 20660.0);
+        driveMotor.config_IntegralZone(0, 300.0);
+        driveMotor.configPeakOutputForward(1.0);
+        driveMotor.configPeakOutputReverse(-1.0);
+        driveMotor.configClosedloopRamp(0.3);
+        driveMotor.configOpenloopRamp(0.3);
```

Comment:
I added in setters for both closed and open loop ramp rate, just to be on the safe side.